ibm/service/power/data_source_ibm_pi_workspace.go

```diff
@@ -40,6 +40,11 @@ func DatasourceIBMPIWorkspace() *schema.Resource {
 				Description: "Workspace information.",
 				Type:        schema.TypeMap,
 			},
+			Attr_PowerEdgeRouterStatus: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "PER status",
+			},
```

Comment:
This is a map

``  "powerEdgeRouter":{
         "type": "automated" | "manual",
         "state": "active" | "error" | "warning" | "configuring" | "removing" | "inactive",
         "migrationStatus": "initializing" | "migrating " | "deleted" | "complete"
     }``